superbench/benchmarks/micro_benchmarks/dist_inference_cpp/dist_inference.cu

```diff
@@ -60,6 +61,9 @@ using cublasLtHalf = hipblasLtHalf;
 #else
 #define DIST_INF_HIP_COMPUTETYPE_F32 HIPBLASLT_COMPUTE_F32
 #endif
+#if defined(USE_HIPBLASLT_TUNING)
```

Comment:
The same question: does it affect the performance?

Reply:
It's not supported by all ROCm versions, see above.

Reply:
Revised.